### PR TITLE
Base elements

### PR DESCRIPTION
**Pull request by alex-windett**
Creating elements like h1, h2, h3 from an object instead of creating separate files for each

### CODE REVIEW

**Comment by alex-windett** [comment]
@yldio/joyent-technical Can someone check this - its seems slightly messy but all tests are passing

File: package.json [hunk 1]
```diff
@@ -18,7 +18,8 @@
     "prepush": "make test"
   },
   "dependencies": {
-    "husky": "^0.11.9"
+    "husky": "^0.11.9",
+    "react-faux-dom": "^3.0.0"
```
Comment (tomgco) [inline package.json]: This shouldn't be in here, I would run a `yarn remove react-faux-dom`

File: ui/stories/index.js [hunk 2]
```diff
@@ -8,6 +8,7 @@ const {
 const {
   Base,
   Column,
+  Topology,
```
Comment (tomgco) [inline ui/stories/index.js]: This is a merge conflict and should be removed

File: ui/stories/index.js [hunk 3]
```diff
@@ -109,3 +110,14 @@ storiesOf('Metrics', module)
       </Row>
     </Base>
   ));
+
```
Comment (tomgco) [inline ui/stories/index.js]: Same again

File: ui/stories/services.js [hunk 4]
```diff
@@ -0,0 +1,132 @@
+module.exports = {
```
Comment (tomgco) [inline ui/stories/services.js]: Same again

File: yarn.lock [hunk 5]
```diff
@@ -74,6 +74,17 @@ process-nextick-args@~1.0.6:
   version "1.0.7"
   resolved "https://registry.yarnpkg.com/process-nextick-args/-/process-nextick-args-1.0.7.tgz#150e20b756590ad3f91093f25a4f2ad8bff30ba3"
 
+query-selector@^1.0.9:
+  version "1.0.9"
+  resolved "https://registry.yarnpkg.com/query-selector/-/query-selector-1.0.9.tgz#917fd31b7379b53fd441e536af647552e01e7e9e"
+
+react-faux-dom@^3.0.0:
```
Comment (tomgco) [inline yarn.lock]: This should be deleted as per the other yarn files.

**Comment by alex-windett** [comment]
Re-created in https://github.com/yldio/joyent-portal/pull/159

**Pull request closed without merging**